aspnet/security/introduction-to-aspnet-identity.rst

```diff
@@ -0,0 +1,100 @@
+
+Getting started with ASP.NET Identity
+-------------------------------------
+ 
+ASP.NET Identity is used in the Visual Studio project templates for ASP.NET `MVC <http://www.asp.net/mvc>`_, `Web Forms <http://www.asp.net/web-forms>`_, `Web API <http://www.asp.net/web-api>`_ and `SPA <http://www.asp.net/single-page-application>`_. In this walkthrough, we'll illustrate how the ASP.NET 5 project templates use ASP.NET Identity to add functionality to register, log in and log out a user. 
+
+The purpose of this article is to give you a high level overview of ASP.NET Identity; you can follow it step by step or just read the details. For more detailed instructions about creating apps using ASP.NET Identity, see the Next Steps section at the end of this article.
+
+1. Create an ASP.NET MVC application with Individual Accounts. You can use ASP.NET Identity in an ASP.NET MVC application, as well as other ASP.NET frameworks, such as Web Forms, Web API, SignalR, etc. In this article, you will start with an ASP.NET MVC application. In Visual Studio, select **File** -> **New** -> **Project**. Then, select the **ASP.NET Web Appication** from the **New Project** dialog box. Continue by selecting a **Web Appication** with **Individual User Accounts** as the authenication method.
+
+	.. image:: introduction-to-aspnet-identity/_static/01-mvc.png
+	
+2. The created project contains the following ASP.NET Identity package.
+
+- ``Microsoft.AspNet.Identity.EntityFramework``
+ The `Microsoft.AspNet.Identity.EntityFramework <http://www.nuget.org/packages/Microsoft.AspNet.Identity.EntityFramework/>`_ package has the Entity Framework implementation of ASP.NET Identity which will persist the ASP.NET Identity data and schema to SQL Server. 
+
+.. note:: In Visual Studio, you can view NuGet packages details by selecting **Tools** -> **NuGet Package Manager** -> **Manage NuGet Packages for Solution**.
+ 
+3. Creating a user.
+
+ Launch the application from Visual Studio (**Debug** -> **Start Debugging**) and then click on the **Register** link in the browser to create a user. The following image shows the *Register* page which collects the user name and password.
+
+	.. image:: introduction-to-aspnet-identity/_static/02-reg.png
+
+When the user clicks the **Register** button, the ``Register`` action of the ``AccountController`` creates the user by calling ``CreateAsync`` function of the ``UserManager`` object, as shown below:
+
+.. literalinclude:: introduction-to-aspnet-identity/sample/src/ASPNET-IdentityDemo/Controllers/AccountController.cs
+	:language: c#
+	:lines: 101-127
+	:emphasize-lines: 4,10
+	:dedent: 8
+	
+4. Log in.
+ If the user was successfully created, they are logged in by the ``SignInAsync`` method, also contained in the ``Register`` action.
```

Comment:
Maybe talk about what "sign in" actually means in this context - I think it's basically storing a cookie with the user claims?